generator/osm2type.cpp

```diff
@@ -1044,7 +1044,19 @@ void GetNameAndType(OsmElement * p, FeatureBuilderParams & params,
   // Stage5: Postprocess feature types.
   PostprocessElement(p, params);
 
-  params.FinishAddingTypes();
+  {
+    std::string const typesString = params.PrintTypes();
+
+    // Unknown type is possible in unit tests.
+    std::string const id = (p->m_type != OsmElement::EntityType::Unknown) ?
+          DebugPrint(GetGeoObjectId(*p)) : std::string("Unknown");
+
+    if (params.RemoveInconsistentTypes())
+      LOG(LWARNING, ("Inconsistent types for:", id, "Types:", typesString));
+
+    if (params.FinishAddingTypesEx() == FeatureParams::TYPES_EXCEED_MAX)
+      LOG(LWARNING, ("Exceeded types count for:", id, "Types:", typesString));
```

Comment:
Leads to double logging, e.g.
```
LOG TID(3) WARNING   20.2459 indexer/feature_data.cpp:456 FinishAddingTypesEx(): Exceeded max types count: amenity-recycling-container recycling-cans recycling-cardboard recycling-glass_bottles recycling-green_waste recycling-paper recycling-plastic recycling-plastic_bottles recycling-scrap_metal
LOG TID(3) WARNING   20.2461 generator/osm2type.cpp:1323 GetNameAndType(): Exceeded types count for: Osm Node 5264249349 Types: amenity-recycling-container recycling-cans recycling-cardboard recycling-glass_bottles recycling-green_waste recycling-paper recycling-plastic recycling-plastic_bottles recycling-scrap_metal
```

Reply:
Please add total types count, like it was before (otherwise have to count manually for each line - very inconvenient).
`LOG(LWARNING, ("Exceeded max types count, got total", m_types.size(), ":", TypesToString(m_types)));`